src/Footer.jsx

```diff
@@ -196,6 +196,49 @@ export const Footer = ({
 	);
 };
 
+Footer.defaultProps = {
+	createMeetup: {
+		text: 'Create a Meetup',
+		link: '/create/'
+	},
+	linkSets: [
+		{
+			header: 'Your Account',
+			items: [
+				<a href='/register'>Settings</a>,
+				<a href="/logout">Log out</a>,
```

Comment:
nitpick - some of these `href`s here and below are using `""` and some are using `''`

Reply:
Woops. I'll clean up...